src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -92,17 +92,26 @@ private void bindListeners(ReadOnlyPerson person) {
      */
     private void initPhoto(ReadOnlyPerson person) {
         try {
-            if (person.getPhoto() != null) {
-                File photoFile = new File(person.getPhoto().getFullPhotoDir());
+            File photoFile = new File(person.getPhoto().getFullPhotoDir());
+            if (photoFile.exists()) {
                 FileInputStream fileStream = new FileInputStream(photoFile);
                 Image personPhoto = new Image(fileStream);
                 photo = new ImageView(personPhoto);
                 photo.setFitHeight(person.getPhoto().HEIGHT);
                 photo.setFitWidth(person.getPhoto().WIDTH);
                 cardPane.getChildren().add(photo);
+            } else {
+                File defaultPhotoFile = new File(person.getPhoto().getTemplatePhotoDir());
+                FileInputStream defaultFileStream = new FileInputStream(defaultPhotoFile);
+                Image defaultPersonPhoto = new Image(defaultFileStream);
+                photo = new ImageView(defaultPersonPhoto);
+
+                photo.setFitHeight(person.getPhoto().HEIGHT);
+                photo.setFitWidth(person.getPhoto().WIDTH);
+                cardPane.getChildren().add(photo);
```

Comment:
Should the user also be informed that the photo was not found when having to use a default photo?